feature/gribi/otg_tests/encap_frr_with_reencap_vrf_test/encap_frr_with_reencap_vrf_test.go

```diff
@@ -249,13 +253,6 @@ func dutInterface(p *ondatra.Port, dut *ondatra.DUTDevice) *oc.Interface {
 		i.Enabled = ygot.Bool(true)
 	}
 
-	if p.PMD() == ondatra.PMD100GBASEFR {
```

Comment:
removing this is causing failures for Arista, if it is not needed, please put it in a deviation for Juniper

Reply:
Moved the code for explicit configuration of port speed under the deviation. Please check.